R/LSTcentralLimitTheorem.R

```diff
@@ -56,70 +70,137 @@ LSTcentralLimitTheorem <- function(jaspResults, dataset, options) {
                               "cltSkewDirection",
                               "cltSkewIntensity"))
   pdPlot$plotObject <- .distributionPlotFunction(distribution = distribution, mean = mean, sd = sd, range = range,
-                                                 skew = skew, skewDirection = skewDirection, fillColor = "coral")
+                                                 skew = skew, prob = prob, skewDirection = skewDirection,
+                                                 fillColor = "coral", showMean = showMean)
   return(pdPlot)
 }
 
 
-.cltTakeSamples <- function(jaspResults, options, data) {
+.cltTakeSamples <- function(jaspResults, options, data, replace = TRUE) {
   n <- options[["cltSampleSize"]]
   k <- options[["cltSampleAmount"]]
   sampleList <- list()
-  for (i in 1:k) {
-    sampleList[[i]] <- sample(x = data[["x"]], size = n, replace = TRUE)
+  if (replace) {
+    for (i in 1:k) {
```

Comment:
`1:k` -> `seq_len(k)`

Reply:
Ah, I will change this everywhere. Very useful, thanks!